config/environments/production.rb

```diff
@@ -102,11 +98,14 @@
   # contains parameters (excluding password) that can
   # help with debugging
   #
+  config.lograge.formatter = LogFormatter.new
   config.lograge.enabled = true
   config.lograge.custom_payload do |controller|
     {
       host: controller.request.host,
-      user_id: controller.current_user.try(:id)
+      user_id: controller.current_user.try(:id),
```

Comment:
oh, maybe the `nil` returns were to help these!

Reply:
Yep, that was why :grin: